system/semodule.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+
+# (c) 2016, Joshua Heilman <joshuah82@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: semodule
+short_description: Manage SELinux policy modules.
+description:
+     - Manage SELinux policy modules.
+version_added: '2.1'
+options:
+  name:
+    description:
+      - Name of the module to configure
+    required: true
+  path:
+    description:
+      - Path to the module package
+      - (required I(state) is set to C(present))
+    required: false
+    default: null
+  state:
+    description:
+      - Whether a module should be C(enabled), C(disabled), installed (C(present)) or removed (C(absent)).
+    required: true
+    choices: [ 'enabled', 'disabled', 'present', 'absent' ]
```

Comment:
i would leave state=present|absent and enabled=yes|no as a different option (to allow installing as disabled)